webpack/components/extensions/HostDetails/Tabs/ErrataTab/ErrataTab.js

```diff
@@ -228,7 +229,7 @@ export const ErrataTab = () => {
   });
 
   const installErrataAction = () => installErrata({
-    hostname, search: fetchBulkParams(),
+    hostname, search: fetchBulkParams({}),
```

Comment:
The need to pass `{}` here is eliminated in https://github.com/theforeman/foreman/pull/9878, but I suppose it's harmless either way. If that gets merged first, you could remove all of these.

Reply:
Agreed!. May be I should wait for your PR before merging